void-linux/void-packages
# pinebookpro: kernel: update to 6.1.9, base: drop firmware dependency, use ALSA UCM

~~Testing and review needed!~~

pinebookpro-base: drop firmware dep, use ALSA UCM

* Pinebook Pro kernel no longer supports video output via altmode dp, and
that was the only reason to install the linux-firmware dependency.
* Replace the asound.state file with a dependency on ALSA UCM, which now
supports the Pinebook Pro.

pinebookpro-kernel: update to 6.1.9

* Adopt.
* Bump to new LTS branch.
* Remove merged or obsolete DTS patches.
* Remove merged gamma LUT patchset.
* Remove hacky USB Type-C alternate mode DisplayPort patchset.
* Use zstd to compress modules

[ci skip]

[skip ci]



#### Testing the changes
- I tested the changes in this PR: **YES**




#### Local build testing
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64 (cross)
  - aarch64-musl (cross)
